BUILD

```diff
@@ -52,9 +52,10 @@ cc_library(
         "include/cctz/zone_info_source.h",
     ],
     includes = ["include"],
-    # Note that OS X and iOS both have a dependency on Foundation but it is no
-    # longer explicitly added as a linkopt as that causes problems cross-compiling android
-    # builds on OSX, and bazel will add it automatically.
+    # Note that OS X and iOS both have a dependency on CoreFoundation but it is
+    # no longer explicitly added as a linkopt as that causes problems
+    # cross-compiling android builds on OSX, and bazel will add it
+    # automatically.
```

Comment:
OK, thanks.  If you have the stamina for one more edit, though, I'd suggest ...
```
    # OS X and iOS no longer use `linkopts = ["-framework CoreFoundation"]`
    # as (1) bazel adds it automatically, and (2) it caused problems when
    # cross-compiling for Android.
```

Reply:
SGTM!  I have no preference on phrasing so happy to land whatever works for you folks